benchmark/swebench-rna-first/issue825/rna_query.py

```
@@ -0,0 +1,288 @@
+            raise ValueError(f"operational_cache_changed:{relative}")
+        members.append({"path": relative, "bytes": after.st_size, "sha256": digest})
+    if not members:
+        raise ValueError("operational_cache_empty")
+    return sha(canonical({
+        "schema_version": "issue825-operational-cache-inventory-v1",
+        "members": members,
+    }))
+
+
+def git(repo: Path, *args: str) -> str:
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)